app/views/schools/dashboard/show.html.erb

```diff
@@ -5,43 +5,85 @@
 <% end %>
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
+  <div class="govuk-grid-column-three-quarters">
     <span class="govuk-caption-l"><%= @school.name %></span>
     <h1 class="govuk-heading-xl govuk-!-margin-bottom-7">Manage your training</h1>
 
-    <p class="govuk-body-l govuk-!-margin-bottom-5">
-      Check and complete steps for every academic year/cohort
-    </p>
-
-    <h2 class="govuk-heading-m govuk-!-margin-bottom-1">Your cohorts</h2>
-
-    <table class="govuk-table">
-      <thead class="govuk-table__head">
-        <tr class="govuk-table__row">
-          <th class="govuk-table__header" scope="col">Year</th>
-          <th class="govuk-table__header" scope="col">Programme</th>
-          <th class="govuk-table__header" scope="col">Status</th>
-        </tr>
-      </thead>
-      <tbody class="govuk-table__body">
-        <% @school_cohorts.each do |school_cohort| %>
+    <% @school_cohorts.each do |school_cohort| %>
+
+      <% if school_cohort.core_induction_programme? || school_cohort.full_induction_programme? %>
+        <p class="govuk-body govuk-!-margin-bottom-5">
+          <%= govuk_link_to 'Add your early career teacher and mentor details', schools_participants_path(cohort_id: school_cohort.cohort.start_year) %>
```

Comment:
Checked with @Lloyd-K , this link text should be conditional. The "Add" should be omitted once the first participant has been added

Reply:
That would make more sense. What about adding 'View...' instead of 'Add...' @Lloyd-K? - I think having a verb helps. I'll update the ticket with this as an additional acceptance criteria

Reply:
@ericaporter - the link allows users to add, edit or view, so we've used 'Your ...' to cover all bases.